### ISSUE TITLE
Number of class instances does not increase with training parameter `augment=True`

### ISSUE THREAD

**Issue by fninsiima** [ultralytics/ultralytics#18796]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I was recently testing out the augmentation parameter `augment` to increase my training samples.
I did this via `augment=True`

```
model.train(data='/content/data.yaml', epochs=50, imgsz=800, augment=True)
```

I observed the output but noticed that my instances did not increase.
Is this normal functionality?

<img width="514" alt="Image" src="https://github.com/user-attachments/assets/77644c83-10c8-4305-83fe-5048638de63a" />

**Comment by UltralyticsAssistant** [comment]
👋 Hello @fninsiima, thank you for bringing this up and for using Ultralytics 🚀! We recommend checking out our [Docs](https://docs.ultralytics.com/) for more insights and examples on training and augmentation. You can also find useful specifics in the [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage guides.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) that demonstrates the issue you're experiencing. This can help us debug more efficiently. Additionally:
- Confirm the `ultralytics` version you're using by running `pip show ultralytics`.
- Share the complete command/configuration you used and any relevant logs or outputs.

If this is a ❓ Question or if you're exploring augmentation behavior, please give additional context—examples of your data and what specific behavior you are observing, compared to your expectations.

### Augmentation information
`augment=True` applies data augmentation during training, but note that this **does not increase your dataset's instance count** during training; it only applies transformations to existing images dynamically at training time. Let us know if further clarification is needed!

### Community Support
If you'd like more immediate help, feel free to join our Ultralytics community:
- Real-time chat: [Discord](https://discord.com/invite/ultralytics) 🎧
- In-depth discussions: [Discourse](https://community.ultralytics.com/)
- Community sharing: [Subreddit](https://reddit.com/r/Ultralytics)

### Upgrade
Ensure you're using the latest `ultralytics` package to benefit from recent features and fixes. Upgrade it here:

```bash
pip install -U ultralytics
```

### Environments
YOLO models can also be run on several verified environments:
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/).
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/).
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>.

### Status
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

This is an automated response to help expedite your issue 🎉. An Ultralytics engineer will review your issue and provide further assistance soon—stay tuned!

**Comment by Y-T-G** [comment]
1. `augment` argument doesn't control augmentations. It's for TTA during prediction. It's not meant to be used for training. Valid arguments are mentioned in the docs for training. https://docs.ultralytics.com/modes/train/#train-settings
2. Your screenshot is not of the training set. It's of the validation set and is showing the number of images and instances in validation set.
3. Augmentations are enabled by default for training in Ultralytics and they're applied on-the-fly to an image. It doesn't increase the number of times an image appears in an epoch, so the number of images would remain the same. Each image appears once and augmentation may or may not be applied on that image based on the randomness.
